### ISSUE TITLE
Dependency on imdone-core non-existing version

### ISSUE THREAD

**Issue by krtek** [imdone/imdone-atom#33]
Installation fails with following error:

```
npm ERR! Darwin 15.0.0
npm ERR! argv "/Applications/Atom.app/Contents/Resources/app/apm/bin/node" "/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/Users/krtek/.atom/.apm/.apmrc" "--userconfig" "/Users/krtek/.atom/.apmrc" "install" "/private/var/folders/bl/h8tpp3n90zd3grk7vd6p9dm00000gn/T/d-115830-5916-wjzrm1/package.tgz" "--target=0.30.7" "--arch=x64"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3

npm ERR! version not found: imdone-core@1.2.23
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /private/var/folders/bl/h8tpp3n90zd3grk7vd6p9dm00000gn/T/apm-install-dir-115830-5916-1g027yg/npm-debug.log
```

The issue seems to be that **imdone-core** last released version is `1.2.9` but `package.json` states dependency on `1.2.23`?

**Comment by piascikj** [comment]
That will teach me to push a change and then leave for the day.  I just published imdone-core 1.2.23.  The install should now work.

**Comment by krtek** [comment]
Thank you!
